### PR TITLE
octodns commands

### PR DESCRIPTION
**Pull request by eyenx**
This creates 5 new packages, as octodns has 5 different commands/binaries.

I didn't know any better and implemented it with different packages, is there a better option to do this?

### CODE REVIEW

**Comment by eyenx** [comment]
@bfirsh Is the project still alive?

**Comment by tjamet** [comment]
@eyenx Yes, I am contributing as maintainer so far, though I didn't go on the packages part yet
Will do it sometime soon

**Comment by eyenx** [comment]
@tjamet any updates?

**Comment by tjamet** [comment]
I understand the base image is built from https://github.com/eyenx/docker-octodns/blob/master/Dockerfile correct?

This leads me to 2 questions:
- What is the benefit we get having an indirection regarding the image?
- Would it be possible to use python 3 instead?

**Comment by eyenx** [comment]
> I understand the base image is built from https://github.com/eyenx/docker-octodns/blob/master/Dockerfile correct?
 yes
> This leads me to 2 questions:
> 
>     * What is the benefit we get having an indirection regarding the image?
I'm using that image for some ci/cd pipelines and not specific to `whalebrew`
>     * Would it be possible to use python 3 instead?
I'll try.

**Comment by eyenx** [comment]
@tjamet is now using `python3`

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -62,6 +62,7 @@ jobs:
             - nmap
             - node
             - npm
+            - oc
```
Comment (tjamet) [inline .github/workflows/push.yml:65]: isn't it `octodns-*` ?

Comment (eyenx) [reply .github/workflows/push.yml]: oh man, monday morning...

**Pull request merged**